servicebus/data-plane/.gitattributes

```diff
@@ -0,0 +1,40 @@
+###############################################################################
```

Comment:
This isn't necessary. The values at the root of the repo overlap this.

Reply:
Will remove it. This PR is only focus on POM files changes after Service bus SDK lib migration: no changes #3428. Will consider your valuable suggestion in the one of next PRs.

Reply:
This PR will be more clear after Wes merged the  #3428 PR.

Reply:
For future reference, it would make sense to ask for review of a PR only when it it's ready for review, to enable focused feedback and save everyone's time.